app/assets/javascripts/Components/Result/marks_panel.jsx

```diff
@@ -219,7 +233,25 @@ class CheckboxCriterionInput extends React.Component {
   }
 }
 
-class FlexibleCriterionInput extends React.Component {
+CheckboxCriterionInput.propTypes = {
+  id: PropTypes.number.isRequired,
+  max_mark: PropTypes.number,
+  mark: PropTypes.number,
+
+  expanded: PropTypes.bool.isRequired,
+  unassigned: PropTypes.bool.isRequired,
+  released_to_students: PropTypes.bool.isRequired,
+
+  oldMark: PropTypes.oneOfType([PropTypes.object, PropTypes.array]),
```

Comment:
This should just be `PropTypes.object` (for all three input components). You should check how you're settings these values in the tests.